ecommerce/frontend/src/components/product/ProductHeader.js

```diff
@@ -0,0 +1,92 @@
+import './ProductHeader.less'
+
+import { HeartOutlined, ShoppingCartOutlined } from '@ant-design/icons'
+import { Button, Modal, Skeleton } from 'antd'
+import React, { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import { ProductImageGallery } from './ProductImageGallery'
+import { ProductExtra } from './ProductExtra'
+import { ChooseListModal } from '../ChooseListModal'
+
+export const ProductHeader = ({ product, loading = false }) => {
+    const [isChooseListModalVisible, setIsChooseListModalVisible] = useState(false)
+
+    const onAddToCartClicked = () => {
+        console.log('add to cart', product)
```

Comment:
This can be implemented since it's already linked to a working endpoint.

Reply:
Oh right, I will do that then.